draft-irtf-cfrg-opaque.md

```diff
@@ -754,40 +751,38 @@ Steps:
 3. Output (request, r)
 ~~~
 
-#### CreateCredentialResponse(request, pkS, kU, envU, pkU)
+#### CreateCredentialResponse(request, kU, envU)
 
 ~~~
-CreateCredentialResponse(request, pkS, kU, envU, pkU)
+CreateCredentialResponse(request, kU, envU)
 
 Input:
 - request, a CredentialRequest structure
-- pkS, public key of the server
 - kU, OPRF key associated with idU
 - envU, Envelope associated with idU
-- pkU, Public key associated with idU
 
 Output:
 - response, a CredentialResponse structure
 
 Steps:
 1. Z = Evaluate(kU, request.data)
-2. Create CredentialResponse response with (Z, pkS, envU)
+2. Create CredentialResponse response with (Z, envU)
 3. Output response
 ~~~
 
 #### RecoverCredentials(pwdU, blind, request, response)
 
 ~~~
-RecoverCredentials(pwdU, blind, request, response)
+RecoverCredentials(pwdU, cleartext_creds, blind, response)
```

Comment:
Line 806 should output an instance of `Credentials`, then, right? (Currently it outputs a secret_credentials)

Reply:
Hmm, it is nice if it just returns a `Credentials` type. I'll make the change!